user_guide.md

```diff
@@ -300,6 +300,29 @@ rootCmd.PersistentFlags().StringVarP(&Region, "region", "r", "", "AWS region (re
 rootCmd.MarkPersistentFlagRequired("region")
 ```
 
+## Flag Groups
```

Comment:
Should this be within the "Working with flags" section like the "Required flags" section above, or did you want it in a standalone section?  If it should be like "Required flags" it should be prefixed with a triple `###` instead of a double.

Reply:
Thanks; I think that was just a typo, I meant to have it be at the same level as the required flags section.